keps/sig-network/3453-minimize-iptables-restore/kep.yaml

```diff
@@ -3,7 +3,7 @@ kep-number: 3453
 authors:
   - "@danwinship"
 owning-sig: sig-network
-status: implementable
+status: implemented
```

Comment:
We're not changing to implemented, until the actual release where the feature is GA is out.